runtime/doc/pi_netrw.txt

```diff
@@ -1,12 +1,12 @@
-*pi_netrw.txt*  Nvim
+*pi_netrw.txt*  For Vim version 8.1.  Last change: 2019 Jul 17
```

Comment:
Why is this line changed in this PR?

Reply:
Why not? Likely I was intending to bring netrw closer to upstream.

Reply:
Shouldn't an exception be added to `vim-patch.sh` to keep this line in diff then?

Reply:
The less we do in service of netrw, the better. I wouldn't favor a netrw-specific code in vim-patch.sh unless it results in a net reduction of netrw-related work :)

Reply:
If no exception is added to `vim-patch.sh`, this line almost never gets updated, and is constantly out-of-date. Currently in Nvim it's:
```
*pi_netrw.txt*  For Vim version 8.2.  Last change: 2020 Aug 15
```
While in Vim it's:
```
*pi_netrw.txt*  For Vim version 9.1.  Last change: 2024 Nov 02
```
Even though the remaining parts of `pi_netrw.txt` is almost the same.

So there are two options:
1. Remove the date
2. Add an exception to `vim-patch.sh` so that the date update in `pi_netrw.txt` is no longer ignored.

Reply:
I'd lean towards removing the date.